### DISCUSSION TITLE
Devices not visible in Home App

### DISCUSSION THREAD

**Discussion by Laotu77** [thoukydides/homebridge-homeconnect#215] [Q&A]
Hi,
I configured the plugin according to the instructions, the service is started in HB but I cannot find any device in the Home App.
How are the usually added?, should they appear automatically under the "Add new device" section or is a code required?
Thx,
Laotu77

**Comment by thoukydides** [comment]
If you have correctly configured this plugin the it will automatically add all of the appliances in your Home Connect account. This will be obvious from the Homebridge log.

The only required step within the Apple Home app is to add the Homebridge bridge. If this is the first time that you have used Homebridge then:
- Open the Home app on your iPhone/iPad.
- Tap the Home tab, then tap on the `+` icon.
- Tap **Add Accessory**, then scan the QR code shown in the Homebridge UI or your Homebridge logs.

Are you using Homebridge UI or setting up Homebridge from the command line?

**Reply by Laotu77** [reply]
Ah, beginner's mistake. I had added the Homebridge hub, but I wasn't checking under the hub settings for the devices. They were already there but didn't show up in the overview of the home.
Now, I will look into alternative apps... because as you rightfully stated, there are limitations, e.g. 20 times the same button.
Thank you!
